tests/src/test/java/alluxio/cli/fs/command/LoadCommandIntegrationTest.java

```diff
@@ -59,5 +60,17 @@ public void loadFile() throws IOException, AlluxioException {
     mFsShell.run("load", "/testFile");
     status = mFileSystem.getStatus(uri);
     assertTrue(status.getInAlluxioPercentage() == 100);
+    // Testing loading a file has been loaded fully
+    mFsShell.run("load", "--local", "/testFile");
+    Assert.assertEquals("/testFile" + " loaded" + "\n" + "/testFile" + " loaded" + "\n",
+        mOutput.toString());
+    // Testing "load --local" works when the file isn't already loaded
```

Comment:
please break this to a separate test since it's independent of everything that comes before

Reply:
Done.